### PR TITLE
Send metrics to datadog

### PR DESCRIPTION
**Pull request by pksunkara**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -28,10 +82,32 @@ exports.handler = function (event, context, callback) {
 
     var data = JSON.parse(result.toString('utf8'));
 
+    var metricRegex = /^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z)\ -\ info:\ ([a-z]+):.*?(metric#.*)+$/;
```
Comment (w-vi) [inline index.js:85]: Do we really need to match the time that exactly? Are we using it somewhere? I think we can make a more generic and thus faster regex.

Comment (pksunkara) [inline index.js:85]: Yup, we are using it for timestamp for the data being sent.

File: Makefile [hunk 2]
```diff
@@ -3,6 +3,7 @@ PROGRAM ?= default
 LOG_GROUP ?= log_group_name
 HOST ?= logs.papertrailapp.com
 PORT ?= 1234
+DATADOG ?= 1234567890abcdef1234567890abcdef12345678
```
Comment (kylef) [inline Makefile:6]: Is this a real API key?

Comment (pksunkara) [inline Makefile:6]: nope. It's just 12345...def

**Pull request merged**